bpytop.py

```diff
@@ -3193,6 +3198,9 @@ def _get_nics(cls):
 			cls.nics.append(nic)
 		if not cls.nics: cls.nics = [""]
 		cls.nic = cls.nics[cls.nic_i]
+		if cls.net_iface: 
```

Comment:
Should probably add a condition to check that `net_iface` is a correct nic name. Something like:
`if cls.net_iface and cls.net_iface in cls.nics:`